README.md

```diff
@@ -90,9 +96,11 @@ Entsprechend muss im Request der Content-Type Header gesetzt werden. Zusätzlich
 |---------------------|------------------|
 | Content-Type        | application/json |
 
-## Beispiel 
+## Beispiele 
 
-### POST Request
+### Query bestesAngebot
+
+#### POST Request
 
     POST https://kex-angebote.kreditsmart.api.europace.de/angebote
     X-Authentication: xxxxxxx
```

Comment:
Können wir hier gleich auf `Authorization: Bearer xxxx` umsteigen?

Reply:
ja, dann sollten wir das in der vorgang-doku auch gleich mitmachen